### ISSUE TITLE
Status disabled

### ISSUE THREAD

**Issue by git2013vb** [don-tnowe/godot-dictionary-inspector#2]
I can't set it as enabled

![image](https://user-images.githubusercontent.com/13235119/210927778-fa3c5633-337d-4907-9adb-235e7b4cc7ee.png)

Its normal?

**Comment by don-tnowe** [comment]
Aaaaare yooooou loading the correct version for your engine?

3.0: [Here](https://github.com/don-tnowe/godot-dictionary-inspector)

4.0: [Here](https://github.com/don-tnowe/godot-dictionary-inspector/tree/godot-4)

**Comment by Spartan322** [comment]
Been getting this issue on Godot 4.0 Beta 10 Mono (haven't checked on non-mono but there shouldn't be any reason the mono version should effect that) installing through the asset lib UI. Only error in the output when I try to enable this plugin is `Transient parent has another exclusive child.`

**Comment by don-tnowe** [comment]
Bizarre. I just tried on a new project on both latest 4.0 versions - the error appeared for me too, and...

... I just opened `plugin.gd` and... saved without changing?

An error appeared related to another script... which I also saved without changing.

But even then, the custom editor appeared on **two of three** dictionaries, which was fixed by just re-enabling the plugin.

I'll look into a workaround.
